app/kuma-dp/pkg/dataplane/envoy/envoy.go

```diff
@@ -156,6 +158,18 @@ func (e *Envoy) Start(stop <-chan struct{}) error {
 	}
 }
 
+func (e *Envoy) DrainConnections() error {
+	resp, err := http.Post(fmt.Sprintf("http://localhost:%d/healthcheck/fail", e.opts.AdminPort), "", nil)
```

Comment:
In `admin_proxy_generator.go` envoy is listening on `127.0.0.1` (not `localhost`), I think it's better to use ipv4 address explicitly